Testscripts/Linux/verify_uio_driver.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+########################################################################
+#
+# Description:
+#       This script was created to verify the uio_hv_generic module.
+#
+#       The test performs the following steps:
+#         1. Assign a NIC to uio_hv_generic driver and check the uio device
+#         2. Restore to hv_netvsc and check the eth1
+#
+########################################################################
+
+# Source utils.sh
+. utils.sh || {
+    echo "ERROR: unable to source utils.sh!"
+    echo "TestAborted" > state.txt
+    exit 0
+}
+
+echo "device_id=$(cat /sys/class/net/eth1/device/device_id|tr -d '{}')" >> constants.sh
+echo "class_id=$(cat /sys/class/net/eth1/device/class_id|tr -d '{}')" >> constants.sh
+
+# Source constants file and initialize most common variables
+UtilsInit
+
+function checkCmdExitStatus ()
```

Comment:
we have check_exit_status in utils.sh, you can use it directly

Reply:
Hi @LiliDeng ,
The check_exit_status can only set Aborted status but not Failed status. Can we update the function? Or just keep using a new function as in my script? Thanks!

Reply:
> I think you will need to check out the test result with other distros. This is how we used to do in the new test case addition. Try with Ubuntu, SLES - you pick up the version.

Hi @juhlee-microsoft ,
Sorry currently we may not have enough knowledge base and energy to debug the case in ubuntu/SLES. But I'll add the distro condition in the cases to prevent failure in other distros. Thank you so much for noticing!